taskiq/cli/scheduler/args.py

```diff
@@ -3,13 +3,14 @@
 from typing import List, Optional, Sequence
 
 from taskiq.cli.common_args import LogLevel
+from taskiq.scheduler.scheduler import TaskiqScheduler
 
 
 @dataclass
 class SchedulerArgs:
     """Arguments for scheduler."""
 
-    scheduler: str
+    scheduler: str | TaskiqScheduler
```

Comment:
Do not use 3.10+ notation. Since taskiq should work even with older versions of python.